guidelines.rst

```diff
@@ -309,3 +404,193 @@ These objects should be type-checked as if they were ``PyObject*``.
     - https://github.com/capi-workgroup/api-evolution/issues/29
     - https://github.com/capi-workgroup/decisions/issues/19
 
+
+Return values
+=============
+
+The return value of a function must indicate whether an exception was set.
+It must not be necessary to use ``PyErr_Occurred`` to disambiguate.
```

Comment:
Should we emphasise this more?

Reply:
I don't think so; the second redundant sentence should be fine.
Since it's a principle that guides the other guidelines, the reader will be fine if they skip it.